xml/src/web-fragment_6_1.xsds

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2009, 2024 Oracle and/or its affiliates and others.
```

Comment:
Is this a correct copyright?

Reply:
I just duplicated the 6_0 version and updated the imports/versions... In that case I would think the copyright history would follow.

Reply:
ah, okay. Thanks for explaining this!